webapp/middleware/searchHashtag.js

```diff
@@ -0,0 +1,16 @@
+import { exec, build } from 'xregexp/xregexp-all.js'
+
+export default async ({ store, env, route, redirect }) => {
+  let publicPages = env.publicPages
+  // only affect non public pages
```

Comment:
sorry, what does this mean? 
by `publicPages` do you mean pages that a user can visit without being signed in?
or is there some meaning for this I don't know about?

Reply:
This I have taken over from `webapp/middleware/authenticated.js`. @mattwr18 
In my understanding `publicPages ` do mean pages that a user can visit without being signed in. Yes.

Reply:
You don't have to check public pages again. The `authenticated` middleware is one of two default middlewares checked for every route, as specified in [nuxt.config.js](https://github.com/Human-Connection/Human-Connection/blob/master/webapp/nuxt.config.js#L126).

I would not use a middleware for sth. that affects only one route, though!